cluster_test.go

```diff
@@ -54,3 +54,11 @@ func TestClusterConfig_translateAddressAndPort_Success(t *testing.T) {
 	assertTrue(t, "translated address", net.ParseIP("10.10.10.10").Equal(newAddr))
 	assertEqual(t, "translated port", 5432, newPort)
 }
+
+func TestClusterConfig_testIdempotancy(t *testing.T) {
+	cfg := NewCluster()
+	assertFalse(t, "cluster config default idempotancy", cfg.DefaultIdempotence)
+	cfg.DefaultIdempotence = true
+	assertTrue(t, "cluster config changed default idempotancy", cfg.DefaultIdempotence)
```

Comment:
This test is really just checking that setting a field to true works and nothing else